### ISSUE TITLE
[SUGGESTION] Add Dockerfile for 24/7 automation

### ISSUE THREAD

**Issue by kernelwernel** [mstephen19/Omegle-Automator#2]
Hi! I really like your project man. Is it possible if you could add this to docker? It would be immensely useful if you could do that :)

**Comment by mstephen19** [comment]
Yo! That's a fantastic suggestion. In the past, I built a bot that runs an automated browser in Node.js that could easily be Dockerized. I don't think Dockerizing this Chrome extension would be practical in most cases. Instead, it would be much more practical to run an automated Firefox/Chrome browser using Playwright. Inputs would be added via a JSON file.

Does this sound like what you have in mind? Thanks for creating an issue btw!

**Comment by kernelwernel** [comment]
> Yo! That's a fantastic suggestion. In the past, I built a bot that runs an automated browser in Node.js that could easily be Dockerized. I don't think Dockerizing this Chrome extension would be practical in most cases. Instead, it would be much more practical to run an automated Firefox/Chrome browser using Playwright. Inputs would be added via a JSON file.
> 
> Does this sound like what you have in mind? Thanks for creating an issue btw!

Yep! I tried to dockerise your code myself but I was unsuccessful at it (web/chrome automation isn't really my strongest skill). But yes, that's what I was thinking about. Adding a Dockerfile would allow people to run it in a VPS or something, which is sort of what I wanna try to do :)

**Comment by mstephen19** [comment]
Would you want this Dockerized version to run off of the Chrome extension? I only ask because the Chrome extension is very UI-focused, but if someone is Dockerizing something, I'd assume they're comfortable with editing a JSON config file.

**Comment by kernelwernel** [comment]
yeah, a json file would be absolutely amazing.
